tests/e2e/lib/repoutils

```diff
@@ -76,3 +76,21 @@ install_qm_rpms() {
 
   dnf install -y bluechi-ctl bluechi-agent bluechi-controller qm hostname
 }
+
+fix_qm_rlimits() {
+  ###########################################################################
+  # Description:                                                            #
+  # Fix: Issue #666 need to set QM rlimit in containers.conf                #
```

Comment:
If this just fix a thing, please just describe what the function do, remove the "fix" word, otherwise it call attention for people fixing something here.

Reply:
If possible just share a github url for an issue that you worked and fixed, how about this example below: 

Function Name: set_qm_rlimit

Description:
This function sets the containers.conf file to set specific resource limits (rlimits) for QM containers, such as nofile (maximum open files). These adjustments ensure the QM environment operates within acceptable system constraints, preventing issues like file descriptor exhaustion or performance bottlenecks in resource-intensive scenarios. Setting these rlimits is critical for maintaining container stability and system reliability, particularly in environments with high concurrency or strict resource policies. 

Reference: htts://github.com/containers/podman/12345

Reply:
Ack